doc/userguide/rules/http-keywords.rst

```diff
@@ -365,11 +410,9 @@ Normalization: leading spaces **are not** part of this buffer. So
 
 Example of the User-Agent header in a HTTP request:
 
-.. image:: http-keywords/user_agent.png
 
 Example of the purpose of ``http.user_agent``:
 
```

Comment:
Missing removal, or missing adding new examples?

Reply:
I've observed a few other cases like this, further down.

Reply:
That's from the request to remove all the images in one commit. The images were removed and the artifacts left so when each keyword is reviewed an example is at least considered again.

Reply:
Fair, better than just removing them and end up forgetting. Thanks for taking this into consideration! :)